### PR TITLE
Fix some of sandandsnow's comments from #94.

### PR DESCRIPTION
**Pull request by jyasskin**
> Re: section 1: “The goal of this document is to define principles that may prove useful in developing technology and policy that relate to privacy and personal data”, it would be helpful if we could elevate the goal to “define privacy principles for Web technologies” (or similar)

Done.

> Re: section 2.2: Perhaps we could leave out the point about data paternalism as the explanation of user agent duties make it clear that is not what is envisaged.

Done, and I removed the reference from the autonomy section.

> Also, is “duty of discretion” the right term?

This comes from [Taking Trust Seriously in Privacy Law](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2655719), which is cited above the list of duties. I've tried to adjust the text to match that definition of discretion better, but @darobin please check that I haven't subverted your intent.

> You might also want to note at the end that “fiduciary” also has legal meaning.

Added. Please check that I said the right thing about the legal meaning.

> Re: section 3: One aspect to keep in mind is that typically in online contexts other parties may be present that are unobservable to the person. At an in-person cocktail party, a person can see who is present and may be monitoring their attributes and actions. Of course, today a cocktail party attendee may not know that the host has a so-called “smart assistant” in listening/observing mode.

Delegated to #97.

> Re: section 4: Would we say burying privacy implications in pages of legalese is a dark pattern or another form of bad behaviour?

We think it is in some cases, but there wasn't any text to add about this.

> Re: section 4.1: We might want to include a statement to the effect that some controls are in practice illusory because the person or user effectively has no choice, accept the terms or don’t use the service.

Delegated to #35.

> Re: section 4.2: I would argue that privacy labour has been increased by a consent-based model for allowing data processing, i.e. where only consent is needed.

Added.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/101.html" title="Last updated on Jan 19, 2022, 5:20 PM UTC (7568fe7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/101/36e9b27...jyasskin:7568fe7.html" title="Last updated on Jan 19, 2022, 5:20 PM UTC (7568fe7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -614,7 +605,8 @@
 Additionally, <dfn data-lt="privacy-labor">privacy labour</dfn> is the practice of having
 a [=person=] carry out the work of ensuring [=data processing=] of which they are the
 subject is [=appropriate=], instead of having the [=parties=] be responsible for that
-work as is more respectable.
+work as is more respectable. Data systems that are based on asking [=people=] for their
```
Comment (samuelweiler) [inline index.html:608]: what can we say instead of "more respectable"?

Comment (samuelweiler) [reply index.html:608]: opened #103

Comment (torgo) [inline index.html:608]:
```suggestion
work. Data systems that are based on asking [=people=] for their
```

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**